### PR TITLE
[CP-1409] Fix disappearing checkboxes on messages

### PR DESCRIPTION
**Pull request by wercix**
Jira: [CP-1409]

**Description**
Bug was fixed, just some fixes after review
-Fix disappearing checkboxes on messages
-Fix the selection manager while deleting thread

Removed useTableSelect hook from thread table
<details>
<summary><b>Screenshots</b></summary>
// put images here
</details>

[CP-1409]: https://appnroll.atlassian.net/browse/CP-1409?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/app/src/messages/components/messages/messages.component.tsx [hunk 1]
```diff
@@ -129,10 +132,11 @@ const Messages: FunctionComponent<MessagesProps> = ({
   const [draftMessage, setDraftMessage] = useState<Message>()
   const [content, setContent] = useState("")
   const debouncedContent = useDebounce(content, 1000)
-  const { selectedRows, allRowsSelected, toggleAll, resetRows, ...rest } =
-    useTableSelect<Thread>(threads)
+  // const { selectedRows, allRowsSelected, toggleAll, resetRows, ...rest } =
```
Comment (igorbezsmertnyi) [inline packages/app/src/messages/components/messages/messages.component.tsx:135]: let's remove it

File: packages/app/src/messages/reducers/messages.reducer.ts [hunk 2]
```diff
@@ -352,5 +358,25 @@ export const messagesReducer = createReducer<MessagesState>(
...
+            selectedItems: { rows: action.payload },
+          }
+        }
+      )
+      .addCase(MessagesEvent.ResetItems, (state) => {
+        return { ...state, selectedItems: { rows: [] } }
+      })
+      .addCase(MessagesEvent.ToggleItem, (state, action: ToggleItemAction) => {
+        const id = action.payload
+        const threadIds = toggleItemSelect(state.selectedItems, id)
```
Comment (igorbezsmertnyi) [inline packages/app/src/messages/reducers/messages.reducer.ts:375]: Please move this logic to the `action`

File: packages/app/src/notification/listeners/outbox-notification.listener.ts [hunk 3]
```diff
@@ -24,6 +24,13 @@ const resourceTypeMap: Record<OutboxEntryType, NotificationResourceType> = {
 
 const newNotifications = (_: any, data: EntryChangesEvent[]): void => {
   data.forEach((item) => {
+    const threadUpdate =
```
Comment (igorbezsmertnyi) [inline packages/app/src/notification/listeners/outbox-notification.listener.ts:27]: Are we still need it?

Comment (wercix) [reply packages/app/src/notification/listeners/outbox-notification.listener.ts]: No, with new managing of checkboxes this is unnecessary, [106999b](https://github.com/mudita/mudita-center/pull/864/commits/106999bfe9a7c55f9033a87e93120de5f3ffffd0)

File: packages/app/src/messages/reducers/messages.reducer.ts (hunk 2 above)
Comment (wercix) [reply packages/app/src/messages/reducers/messages.reducer.ts]: Moved in [106999b](https://github.com/mudita/mudita-center/pull/864/commits/106999bfe9a7c55f9033a87e93120de5f3ffffd0)

File: packages/app/src/messages/components/messages/messages.component.tsx (hunk 1 above)
Comment (wercix) [reply packages/app/src/messages/components/messages/messages.component.tsx]: Removed in [106999b](https://github.com/mudita/mudita-center/pull/864/commits/106999bfe9a7c55f9033a87e93120de5f3ffffd0)

**Review by igorbezsmertnyi — APPROVED** [review]

**Review by dkarski — APPROVED** [review]

**Pull request merged**